spec/request/api/v1/targets/create_spec.rb

```diff
@@ -46,5 +44,37 @@
         expect(response).to have_http_status(:unauthorized)
       end
     end
+
+    context 'when the user already has 9 targets' do
```

Comment:
I think is more appropriate to use the constant `MAX_TARGETS_PER_USER ` here, so you only depend on your constant.
`context 'when the user already has (MAX_TARGETS_PER_USER -1) targets' do` or something like that